models/demos/falcon7b/tests/test_falcon_end_to_end.py

```diff
@@ -294,81 +295,54 @@ def run_test_FalconCausalLM_end_to_end(
 
 
 @pytest.mark.parametrize(
-    "llm_mode, batch, seq_len, kv_cache_len",
+    "llm_mode, num_layers, batch, seq_len, kv_cache_len, model_config_str, expected_output_pcc, expected_k_cache_pcc, expected_v_cache_pcc",
     (
-        ("prefill", 2, 128, 0),
-        ("prefill", 2, 1024, 0),
-        ("decode", 32, 1, 128),
-        ("decode", 32, 1, 1024),
+        ("prefill", 32, 1, 32, 0, "BFLOAT16-L1", 0.97, 0.98, 0.95),
+        ("prefill", 32, 1, 32, 0, "BFLOAT16-DRAM", 0.97, 0.98, 0.95),
+        ("prefill", 32, 1, 128, 0, "BFLOAT16-L1_SHARDED", 0.97, 0.99, 0.96),
+        ("prefill", 32, 1, 1024, 0, "BFLOAT16-L1_SHARDED", 0.98, 0.99, 0.96),
+        ("prefill", 32, 1, 2048, 0, "BFLOAT16-L1_SHARDED", 0.99, 0.99, 0.97),
+        ("decode", 32, 32, 1, 128, "BFLOAT16-DRAM", 0.86, 0.86, 0.86),
+        ("decode", 32, 32, 1, 128, "BFLOAT16-L1", 0.86, 0.86, 0.86),
+        ("decode", 32, 32, 1, 1024, "BFLOAT16-DRAM", 0.86, 0.86, 0.86),
+        ("decode", 32, 32, 1, 1024, "BFLOAT16-L1", 0.86, 0.86, 0.86),
     ),
-    ids=["prefill_seq128", "prefill_seq1024", "decode_batch32", "decode_batch32_1024"],
-)
-@pytest.mark.parametrize(
-    "num_layers, out_pcc, cache_pcc",
-    ((2, 0.98, 0.98), (32, 0.86, 0.86)),
-    ids=["layers_2", "layers_32"],
+    ids=[
+        "prefill_seq32_bf16_l1",
+        "prefill_seq32_bf16_dram",
+        "prefill_seq128_bf16_l1",
+        "prefill_seq1024_bf16_l1",
+        "prefill_seq2048_bf16_l1",
```

Comment:
Can these be renamed to l1sharded if that is the config that is supposed to be used?